src/lib/actions/helpers/decorators/__tests__/decorateHtmlCode.test.js

```diff
@@ -0,0 +1,195 @@
+/***************************************************************************************
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ ****************************************************************************************/
+
+'use strict';
+
+var decorateHtmlCodeInjector = require('inject-loader!../decorateHtmlCode');
+
+describe('decorate html code', function() {
+  var mockTurbine;
+
+  beforeEach(function() {
+    mockTurbine = {
+      replaceTokens: jasmine.createSpy().and.callFake(function(token) {
+        return token.replace(/%(.+?)%/g, function(token, variableName) {
+          return 'replaced - ' + variableName;
+        });
+      })
+    };
+
+    mockTurbineVariable(mockTurbine);
+  });
+
+  afterEach(function() {
+    resetTurbineVariable();
```

Comment:
I think I can make the test pass without the mockTurbineVariable if you want.

Reply:
I guess it would depend on how you do it. Like I've mentioned previously, I would prefer to inject dependencies in the long-term instead of importing them. Until then, this is fine enough.